metadata.yaml

```diff
@@ -44,3 +44,7 @@ provides:
     interface: prometheus_remote_write
   grafana-source:
     interface: grafana_datasource
+
+requires:
+  s3:
```

Comment:
No. We are attempting to keep compatibility with the data platform lib/interfaces

Reply:
I can give my reason for agreeing with "no": The storage suffix does not add any value as s3 is an established term that most who will want to use it will already understand.